NotebookAnalyzers/utils.py

```diff
@@ -27,7 +27,51 @@ def notebookToCode(filename):
     return py_code
 
 def functionsNumber(code):
-    """The function takes a python code sintax tree and returns the number of function definitions"""
+    """The function takes a python code and returns the number of function definitions"""
     tree = ast.parse(code)
     f_num=sum(isinstance(exp, ast.FunctionDef) for exp in tree.body)
-    return f_num
\ No newline at end of file
+    return f_num
+
+def markdownNumber(notebook):
+    """The function takes the notebook dictionary and returns both the number of markdown wors and number of markdown titles"""
+    titles=0
+    markdowns=0
+    for cell in notebook["cells"]:
+        if cell["cell_type"]=='markdown':
+            rows=len(cell['source'])
+            for row in cell['source']:
+                if row[0:2]=='# ':
+                       titles=titles+1
+            markdowns=markdowns+rows
+    return markdowns,titles
```

Comment:
Anche in questo caso applicherei il [single-responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle), costruendo funzioni diverse per calcolare il numero delle parole in Markdown e il numero dei titoli.